lib/modules/manager/sbt/extract.ts

```diff
@@ -285,29 +291,50 @@ const query = q.tree<Ctx>({
 
 export function extractPackageFile(
   content: string,
-  _packageFile: string
+  packageFile: string
 ): PackageFileContent | null {
-  let parsedResult: Ctx | null = null;
-
-  try {
-    parsedResult = scala.query(content, query, {
-      vars: {},
-      deps: [],
-      registryUrls: [REGISTRY_URLS.mavenCentral],
-    });
-  } catch (err) /* istanbul ignore next */ {
-    logger.warn({ err }, 'Sbt parsing error');
-  }
+  if (packageFile === 'project/build.properties') {
+    const groups = sbtVersionRegex.exec(content)?.groups;
+    const sbtVersion = groups?.version;
+    if (sbtVersion) {
+      const sbtDependency: PackageDependency = {
+        datasource: GithubReleasesDatasource.id,
+        depName: 'sbt/sbt',
+        packageName: 'sbt/sbt',
+        versioning: semverVersioning.id,
+        currentValue: sbtVersion,
+        extractVersion: '^v(?<version>\\S+)',
+      };
 
-  if (!parsedResult) {
-    return null;
-  }
+      return {
+        deps: [sbtDependency],
+      };
+    } else {
+      return null;
+    }
+  } else {
+    let parsedResult: Ctx | null = null;
+
+    try {
+      parsedResult = scala.query(content, query, {
+        vars: {},
+        deps: [],
+        registryUrls: [REGISTRY_URLS.mavenCentral],
+      });
+    } catch (err) /* istanbul ignore next */ {
+      logger.warn({ err }, 'Sbt parsing error');
```

Comment:
```suggestion
      logger.warn({ err, packageFile }, 'Sbt parsing error');
```
unrelated but helps for error finding